lms/static/js/courseware/toggle_element_visibility.js

```diff
@@ -1,18 +1,18 @@
 ;(function (define) {
     'use strict';
 
-    define(["jquery"],
-        function ($) {
+    define(['jquery', 'logger', 'moment'],
+        function ($, Logger, moment) {
 
             return function () {
                 // define variables for code legibility
                 var toggleActionElements = $('.toggle-visibility-button');
 
-                var updateToggleActionText = function (targetElement, actionElement) {
+                var updateToggleActionText = function (hideElement, actionElement) {
```

Comment:
Can we rename this variable (both here and where this function is called) to `elementIsHidden`?  The variables ending in `Element` should be elements, it's a little confusing to use this naming convention for booleans as well.